src/components/IconDetail.vue

```diff
@@ -39,6 +39,125 @@ const copy = async(type: string) => {
   }, 2000)
 }
 
+const copyPng = async () => {
+	console.log(color.value);
+	let svg = await getIconSnippet(props.icon, "svg", true, color.value);
+	console.log(svg);
+	let containerDiv = document.createElement("div");
+	containerDiv.innerHTML = svg;
+	//Now we have a DOM element:
+	svg = containerDiv.querySelector("svg");
+	svg.setAttribute("style", `color: ${color.value};`);
+	let data = new XMLSerializer().serializeToString(svg);
+	let svgBlob = new Blob([data], {
+		type: "image/svg+xml;charset=utf-8",
+	});
+	let src = URL.createObjectURL(svgBlob);
+	let image = await toPng(src);
+	console.log(image);
+	navigator.clipboard
+		.write([
+			new ClipboardItem({
+				"image/png": toBlob(image),
+			}),
+		])
+		.then(
+			(...a) => (
+				(copied.value = "Copied"),
+				setTimeout(() => (copied.value = false), 2000)
+			),
+			(...a) => (
+				(copied.value = "Failed to copy"),
+				setTimeout(() => (copied.value = false), 2000)
+			)
+		);
```

Comment:
This will not work on really old browsers, should I use a library to copy instead?

Reply:
Meh, libraries add bloat and dependencies, this works, and people who're using IE11 shouldn't be able to use this project in the first place.